### PR TITLE
Est 46 observacao aba documentacao

### PR DESCRIPTION
**Pull request by amanda-s**
Campo "Observação" na aba "Documentação" na página "Detalhes da inscrição"

### CODE REVIEW

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 1]
```diff
@@ -23,11 +24,29 @@
...
+import br.ufc.quixada.npi.gpa.enums.FinalidadeVeiculo;
+import br.ufc.quixada.npi.gpa.enums.GrauParentesco;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoImovelRural;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoVeiculos;
+import br.ufc.quixada.npi.gpa.enums.MoraCom;
+import br.ufc.quixada.npi.gpa.enums.NivelInstrucao;
+import br.ufc.quixada.npi.gpa.enums.SituacaoImovel;
+import br.ufc.quixada.npi.gpa.enums.SituacaoResidencia;
+import br.ufc.quixada.npi.gpa.enums.TipoEnsinoFundamental;
+import br.ufc.quixada.npi.gpa.enums.TipoEnsinoMedio;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.TipoEnsinoMedio'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l39)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 2]
```diff
@@ -23,11 +24,29 @@
...
 import br.ufc.quixada.npi.gpa.enums.TipoSelecao;
+import br.ufc.quixada.npi.gpa.enums.Turno;
 import br.ufc.quixada.npi.gpa.model.Documento;
+import br.ufc.quixada.npi.gpa.model.HorarioDisponivel;
+import br.ufc.quixada.npi.gpa.model.Inscricao;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.model.Inscricao'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l44)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 3]
```diff
@@ -23,11 +24,29 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import br.ufc.quixada.npi.gpa.enums.DiaUtil;
+import br.ufc.quixada.npi.gpa.enums.Estado;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.Estado'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l29)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 4]
```diff
@@ -23,11 +24,29 @@
...
+import br.ufc.quixada.npi.gpa.enums.FinalidadeVeiculo;
+import br.ufc.quixada.npi.gpa.enums.GrauParentesco;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoImovelRural;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoVeiculos;
+import br.ufc.quixada.npi.gpa.enums.MoraCom;
+import br.ufc.quixada.npi.gpa.enums.NivelInstrucao;
+import br.ufc.quixada.npi.gpa.enums.SituacaoImovel;
+import br.ufc.quixada.npi.gpa.enums.SituacaoResidencia;
+import br.ufc.quixada.npi.gpa.enums.TipoEnsinoFundamental;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.TipoEnsinoFundamental'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l38)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 5]
```diff
@@ -23,11 +24,29 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import br.ufc.quixada.npi.gpa.enums.DiaUtil;
+import br.ufc.quixada.npi.gpa.enums.Estado;
+import br.ufc.quixada.npi.gpa.enums.FinalidadeVeiculo;
+import br.ufc.quixada.npi.gpa.enums.GrauParentesco;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoImovelRural;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoVeiculos;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.GrauParentescoVeiculos'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l33)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 6]
```diff
@@ -23,11 +24,29 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import br.ufc.quixada.npi.gpa.enums.DiaUtil;
+import br.ufc.quixada.npi.gpa.enums.Estado;
+import br.ufc.quixada.npi.gpa.enums.FinalidadeVeiculo;
+import br.ufc.quixada.npi.gpa.enums.GrauParentesco;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoImovelRural;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.GrauParentescoImovelRural'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l32)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 7]
```diff
@@ -23,11 +24,29 @@
...
+import br.ufc.quixada.npi.gpa.enums.SituacaoImovel;
+import br.ufc.quixada.npi.gpa.enums.SituacaoResidencia;
+import br.ufc.quixada.npi.gpa.enums.TipoEnsinoFundamental;
+import br.ufc.quixada.npi.gpa.enums.TipoEnsinoMedio;
 import br.ufc.quixada.npi.gpa.enums.TipoSelecao;
+import br.ufc.quixada.npi.gpa.enums.Turno;
 import br.ufc.quixada.npi.gpa.model.Documento;
+import br.ufc.quixada.npi.gpa.model.HorarioDisponivel;
+import br.ufc.quixada.npi.gpa.model.Inscricao;
+import br.ufc.quixada.npi.gpa.model.PessoaFamilia;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.model.PessoaFamilia'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l45)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 8]
```diff
@@ -43,6 +62,9 @@
 
 	@Inject
 	private ServidorService servidorService;
+	
+	@Inject
+	private InscricaoService inscricaoService;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused private fields such as 'inscricaoService'.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l67)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 9]
```diff
@@ -23,11 +24,29 @@
...
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import br.ufc.quixada.npi.gpa.enums.DiaUtil;
+import br.ufc.quixada.npi.gpa.enums.Estado;
+import br.ufc.quixada.npi.gpa.enums.FinalidadeVeiculo;
+import br.ufc.quixada.npi.gpa.enums.GrauParentesco;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoImovelRural;
+import br.ufc.quixada.npi.gpa.enums.GrauParentescoVeiculos;
+import br.ufc.quixada.npi.gpa.enums.MoraCom;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.MoraCom'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l34)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java [hunk 10]
```diff
@@ -23,11 +24,29 @@
...
+import br.ufc.quixada.npi.gpa.enums.NivelInstrucao;
+import br.ufc.quixada.npi.gpa.enums.SituacaoImovel;
+import br.ufc.quixada.npi.gpa.enums.SituacaoResidencia;
```
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.SituacaoResidencia'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2472371997/issues/source?bid=3244003&fileBranchId=3260965#l37)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 1 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.TipoEnsinoMedio'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l39)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 2 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.model.Inscricao'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l44)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 3 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.Estado'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l29)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 4 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.TipoEnsinoFundamental'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l38)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 5 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.GrauParentescoVeiculos'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l33)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 6 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.GrauParentescoImovelRural'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l32)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 7 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.model.PessoaFamilia'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l45)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 8 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused private fields such as 'inscricaoService'.](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l67)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 9 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.MoraCom'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l34)

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java (hunk 10 above)
Comment (npiuser) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/CoordenadorController.java]: Codacy Issue found: [Avoid unused imports such as 'br.ufc.quixada.npi.gpa.enums.SituacaoResidencia'](https://www.codacy.com/app/NPI-UFC/gpa-assuntos-estudantis/file/2482906807/issues/source?bid=3244003&fileBranchId=3260965#l37)

**Comment by amanda-s** [comment]
Imports não utilizados retirados.

File: gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/ServidorController.java [hunk 11]
```diff
@@ -292,4 +292,17 @@ public String avaliarDocumentacao(@Valid @ModelAttribute("avaliarDocumentacao")
 		redirect.addFlashAttribute("info", MENSAGEM_DE_SUCESSO_AVALIAR_DOCUMENTACAO);
 		return REDIRECT_PAGINA_LISTAR_SELECAO;
 	}
+	
+	@RequestMapping(value= {"adicionarObservacaoDocumentos"}, method = RequestMethod.POST)
+	public String adicionarObsDocumentos(@ModelAttribute("observacaoDocumentos") Inscricao inscricao,
```
Comment (virginiafarias) [inline gpa-mae/src/main/java/br/ufc/quixada/npi/gpa/controller/ServidorController.java]: Já existe um método que faz a avaliação da documentação. Como foi solicitado apenas a inclusão de mais um campo nessa avaliação, o ideal é que o método atual seja modificado pra contemplar esse inclusão.

**Pull request closed without merging**